### PR TITLE
feat(token): add mask network token

### PR DESCRIPTION
**Pull request by developerfred**
### [Mask.io](https://mask.io)

Add Mask Token on Optimistic 

ref: https://github.com/DimensionDev/Maskbook/pull/5284

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1067,6 +1067,28 @@
...
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x69af81e73a73b40adf4f3d4223cd9b1ece623074",
+      "name": "Mask Network",
+      "symbol": "MASK",
+      "decimals": 18,
+      "logoURI": "https://etherscan.io/token/images/mask_32.png",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1077]: Can you add a 256x256 logo to the `/logos` folder in the repo pls and link to that instead

Comment (developerfred) [reply optimism.tokenlist.json:1077]: Yeah

Comment (elenadimitrova) [reply optimism.tokenlist.json]: Thanks but the logo is too big and won't display properly. Can you drop the size to something closer to 256x256?

Comment (elenadimitrova) [reply optimism.tokenlist.json]: @developerfred can you update the logo please so we can merge

Comment (developerfred) [reply optimism.tokenlist.json]: @elenadimitrova I will compress it now.

Comment (developerfred) [reply optimism.tokenlist.json]: done @elenadimitrova

Comment (elenadimitrova) [reply optimism.tokenlist.json]: This looks like a partial image, can you double check 
Screenshot 2022-01-17 at 14 21 26

**Review by elenadimitrova — APPROVED** [review]

**Pull request merged**